collect_app/src/main/res/xml/preferences.xml

```diff
@@ -75,6 +75,12 @@
             android:key="font_size"
             android:title="@string/font_size" />
         <CheckBoxPreference
+            android:id="@+id/instance_sync"
+            android:defaultValue="true"
+            android:key="instance_sync"
```

Comment:
You're talking about something like [this](https://medium.com/@ali.muzaffar/android-sharedpreferences-singleton-to-make-life-easier-f1d802b6cd8e#.sw19gwyxk), right? Seems like a good thing to discuss in the issue @nribeka will file. 😁